src/Symfony/Component/BrowserKit/CookieJar.php

```diff
@@ -68,12 +64,8 @@ public function get($name, $path = '/', $domain = null)
      * You should never use an empty domain, but if you do so,
      * all cookies for the given name/path expire (this behavior
      * ensures a BC behavior with previous versions of Symfony).
-     *
-     * @param string $name   The cookie name
-     * @param string $path   The cookie path
-     * @param string $domain The cookie domain
      */
-    public function expire($name, $path = '/', $domain = null)
+    public function expire(string $name, ?string $path = '/', string $domain = null)
```

Comment:
Since the `$domain` parameter accepts null, the correct typehint here should be `?string`

```suggestion
    public function expire(string $name, ?string $path = '/', ?string $domain = null)
```